airbyte-integrations/connectors/source-freshdesk/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_freshdesk ./source_freshdesk
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=1.0.0
+LABEL io.airbyte.version=1.0.1
```

Comment:
this needs a minor upgrade because changing the schema is not backwards compatible

Reply:
updated